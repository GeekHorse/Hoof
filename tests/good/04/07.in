new right one two three done
delete word
delete word
delete word
new right four five six done
save
