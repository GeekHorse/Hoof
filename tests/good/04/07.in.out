new right four five six done
